src/AspNetCoreModuleV2/RequestHandlerLib/PipeOutputManager.cpp

```diff
@@ -194,13 +194,20 @@ PipeOutputManager::ReadStdErrHandleInternal(
         }
         else if (GetLastError() == ERROR_BROKEN_PIPE)
         {
-            break;
+            return;
         }
     }
-}
 
-VOID
-PipeOutputManager::NotifyStartupComplete()
-{
-    StopOutputRedirection();
+    std::string tempBuffer;
```

Comment:
I would replace `4096` with a constant like `BUFSIZE`.  Also, is there any reason to use `std::string` and `resize()` as opposed to the simpler `char[]`?